### ISSUE TITLE
Issues regarding v3.2

### ISSUE THREAD

**Issue by TontyTon** [henrypp/simplewall#723]
Important issues-

1. Now, I can't add multiple IP addresses/ports at once to user rules by seperating them by semi-colon (;). Have you changed the symbol or removed the option to add multiple IP addresses at once?
If you removed the option then please bring it back, as in v3.2 we have to add each IP address/port one by one.

2. Also, the new UI of Rules Editor require more number of clicks (distribution of options into multiple tabs) for the same work as compared to old versions, so consider bringing back all the options on a single page again.

3. I don't know if this was available earlier or not, but I can't enable/disable notifications for an app in 'Apps' tab of main window. So if I have to re-enable notifications for an app, I have to remove the app from the list, which is not the convenient way.

4. When updating to v3.2, if there was a user rule which contained '*' in remote/local rule, then that rule is 'invalid'. So, have to remove the '*' from all the rules to remove the 'invalid' status. So, v3.2 should ignore '*' after update as there is no need of it now.

Others - These are less important

1. There is no option to add apps in the right click menu in the 'Apps' tab of main window. So, to add an app I have to use option from 'File' menu. I think option to add app in right click menu an in older versions was more convenient.

2. There is no option to allow/block only IPv6 or IPv4 in a user rule, it was also a useful feature, so please also consider bringing the option back.

3. Tool Tip of user rules should show each entry of remote/local rule in new line (as in older versions). It was more convenient, as now have to closely see the IPs/ports whereas earlier each entry was clearly distinguishable.

Great improvements in v3.2 -
1. The new tabs on the main window are really helpful.

2. Thanks for bringing back the blocklist into the UI again.

3. Packets logging is very useful, thanks for adding it too.

**Comment by henrypp** [comment]
>     3\. I don't know if this was available earlier or not, but I can't enable/disable notifications for an app in 'Apps' tab of main window. So if I have to re-enable notifications for an app, I have to remove the app from the list, which is not the convenient way.

`<right click on app item>` -> `Rules` -> `Disable notifications` - it's working as earlier

**Comment by ltGuillaume** [comment]
> There is no option to add apps in the right click menu in the 'Apps' tab of main window. So, to add an app I have to use option from 'File' menu. I think option to add app in right click menu an in older versions was more convenient.

Maybe it would make even more sense to add a small button in the toolbar next to the "Create rule..." button? A context menu should list option that relate to the selected item only.

**Comment by ltGuillaume** [comment]
> 2. Also, the new UI of Rules Editor require more number of clicks (distribution of options into multiple tabs) for the same work as compared to old versions, so consider bringing back all the options on a single page again.

I completely agree. I was relieved to see simplewall's rules editor to _not_ make the same mistake as the Windows Firewall UI, until version 3.2. It's really annoying not to have all the settings available in a single dialog.

**Comment by ltGuillaume** [comment]
> > and the fact that `Ctrl+Tab` doesn't traverse tabs anywhere in simplewall, just like I quoted.
> 
> Yes, `Ctrl+Tab` doesn't switch tabs. Try `Ctrl+1` and `Ctrl+2`. I think i change hotkeys in next version.

`Ctrl+Tab` and `Ctrl+Shift+Tab` work perfectly in v3.6's main window now, thanks! The hotkeys don't yet work in the rules editor.

**Comment by TontyTon** [comment]
@ltGuillaume Now `Escape` key clear the search field, then close the window in v3.6.1. Thanks Henry for quickly solving the issue.

In v3.6.1 too, `Ctrl+tab` or `Ctrl+1/2` don't work in rules editor.

**Comment by ltGuillaume** [comment]
Yes, I saw that, great stuff!
